holoviews/core/util.py

```diff
@@ -2329,3 +2331,30 @@ def flatten(line):
             yield from flatten(element)
         else:
             yield element
+
+
+def lazy_isinstance(obj, class_or_tuple):
```

Comment:
Just tried that in my env and got some weird results.

```python
from holoviews.core.util import lazy_isinstance
import dask.dataframe as dd

ddf = dd.DataFrame({})

print(lazy_isinstance(ddf, 'dask.dataframe:DataFrame'))
print(isinstance(ddf, dd.DataFrame))
```

```
False
True
```

That's because `obj.__module__` is `dask_expr._collection` in this case. Pretty unusual but well!

---

Alternative API with `lazy_instance(obj, module, objname)` where `objname` is either a string or a tuple of strings, e.g. `lazy_instance(obj, 'dask.dataframe', 'DataFrame')`, `lazy_instance(obj, 'cudf', ('DataFrame', 'Series'))`.

Reply:
I have changed it to `.startswith`. This is what is done in hvplot.